theme/gatsby-node.js

```diff
@@ -71,7 +71,7 @@ exports.createPages = async ({graphql, actions}, themeOptions) => {
           // for us here, and does on the first build,
           // but when HMR kicks in the frontmatter is lost.
           // The solution is to include it here explicitly.
-          frontmatter: node.frontmatter
+          // frontmatter: node.frontmatter
```

Comment:
@BinaryMuse I know you added this line to fix the frontmatter in dev, but for some reason `node.frontmatter` wasn't getting any frontmatter except `title`. Commenting this line out fixed the frontmatter for me.

Before I commented out this line: https://doctocat-e2nwgljzt.now.sh/doctocat/components/caption
(Notice that there is no "status" or "view source" link under the heading)

After I commented out this line: https://doctocat-92tbe1p3x.now.sh/doctocat/components/caption
(Notice that the "status" or "view source" link appear under the heading)